packages/Liquid-Tests.package/ObjectRepoTests.class/instance/testifPresentIfAbsentRemote.st

```diff
@@ -0,0 +1,22 @@
+tests
+testifPresentIfAbsentRemote
+	| server port url client presentCheck|
+	presentCheck := false.
+	port := 10000 random.
+	server := RemoteRepoServer new listenOn: port.
+	url := 'http://localhost:' , port.
+	client := RemoteRepoClient new url: url.
+	client
+		at: 'newPoll'
+		ifPresent: [self fail]
+		ifAbsentPut: (LiquidPoll newWithPollDraft: LiquidPollDraft new).
+	client 
+		at: 'newPoll'
+		ifAbsent: [self fail].
+	client 
+		at: 'newPoll'
+		ifPresent: [presentCheck := true]
+		ifAbsentPut: ThrowerOfErrors new. 	
+		
+	self assert: presentCheck equals: true.
+	
```

Comment:
```suggestion
    server destroy
```
[pebble] This makes sure the server doesn't take up a spare port.

Reply:
Thanks for the info. I have included this change with the other changes in the next commit